package.json

```diff
@@ -2,8 +2,8 @@
   "name": "famly",
   "version": "1.0.0",
   "scripts": {
-    "heroku-prebuild": "cd client && yarn add",
-    "client": "cd client && yarn start",
+    "heroku-prebuild": "cd client && yarn",
+    "client": "cd client && yarn run start",
```

Comment:
Please double check if this is correct. I am not sure what the command `yarn` without any arguments like `run` does.

Reply:
It's like yarn install. You can use both ways.